Web Sevice 1 (Soap)/TempratureConverterClient/src/com/metacube/TempratureConverterService.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * TempratureConverterService.java
+ *
+ * This file was auto-generated from WSDL
+ * by the Apache Axis 1.4 Apr 22, 2006 (06:55:48 PDT) WSDL2Java emitter.
+ */
+
+package com.metacube;
+
+public interface TempratureConverterService extends javax.xml.rpc.Service {
+    public java.lang.String getTempratureConverterAddress();
+
+    public com.metacube.TempratureConverter getTempratureConverter() throws javax.xml.rpc.ServiceException;
```

Comment:
If this class exists in same package then I think we do not need to use packagename with class name. Java compiler will find the class by its name also.